.github/workflows/build-ui.yml

```diff
@@ -0,0 +1,53 @@
+name: Build UI
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
+
+      - name: Set up Node environment
+        uses: actions/setup-node@v4.0.2
+        with:
+          node-version: 18.x
+
+      - name: Install dependencies
+        run: npm ci
+        working-directory: ui
+
+      - name: Build and Copy
+        run: npm run buildCopy
+        working-directory: ui
+
+      - id: check_label
+        name: Check PL labels
+        uses: docker://agilepathway/pull-request-label-checker:latest
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Commit static files if dependencies label
+        if: steps.check_label.outputs.label == 'dependencies'
+        run: |
+          git config --global user.email "noreply@github.com"
+          git config --global user.name "dependabot"
```

Comment:
Will the commits looks similar to dependbots commits? 

If yes, then I would suggest to use a different name/email to differentiate between the automated commits by dependbot and our bot.

Reply:
You could also use the github actions bot. 
```
git config user.name 'github-actions[bot]'
git config user.email 'github-actions[bot]@users.noreply.github.com'
```

Not sure but I suspect if you use a username/password that's not a real github user it may show up as "unknown"?